crates/networking/p2p/sync.rs

```diff
@@ -136,63 +177,45 @@ impl SyncManager {
                 // snap-sync: launch tasks to fetch blocks and state in parallel
                 // - Fetch each block's body and its receipt via eth p2p requests
                 // - Fetch the pivot block's state via snap p2p requests
-                // - Execute blocks after the pivote (like in full-sync)
+                // - Execute blocks after the pivot (like in full-sync)
+                let pivot_idx = all_block_hashes
+                    .len()
+                    .checked_sub(MIN_FULL_BLOCKS)
+                    .unwrap_or_default();
```

Comment:
I think this accomplishes the same thing.
```suggestion
                    .saturating_sub(MIN_FULL_BLOCKS);
```

Reply:
Nice catch!